DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -733,20 +733,46 @@ def _buildInputStage(self, simple, generator_input_modules, output_level=None, h
 # MODULE FUNCTIONS GO HERE
 ################################################################################
 
-def sortParameters(parA, parB):
-  """sort the parameters by name: first normal parameters, then set of
-  parameters based on ConfigHelper objects
-  """
-  parTypeA = parA[1]
-  parTypeB = parB[1]
-  if isinstance(parTypeA, ConfigHelper) and isinstance(parTypeB, ConfigHelper):
-    return 1 if str(parA[0]) > str(parB[0]) else -1
-  elif isinstance(parTypeA, ConfigHelper):
-    return 1
-  elif isinstance(parTypeB, ConfigHelper):
-    return -1
+
+def sortParameters(key):
+  if six.PY3:
+    from functools import cmp_to_key
```

Comment:
What's from with the `cmp_to_key` from py2?

Reply:
ahh, I read this was added in python 3.7 :wallb:
https://docs.python.org/2/howto/sorting.html#the-old-way-using-the-cmp-parameter